docs/frontend-system/building-apps/02-built-in-extensions.md

```diff
@@ -0,0 +1,190 @@
+
+#### Inputs
+
+| Name         | Description                | Type                                                                                                                                                   | Optional | Default                                          | Extension creator                                                                                    |
+| ------------ | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- | ------------------------------------------------ | ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| root         | The app root element.      | [coreExtensionData.reactElement](https://backstage.io/docs/reference/frontend-plugin-api.coreextensiondata)                                            | false    | The [`App/Root`](#app-root) extension output.    | No creator available, configure or override the [`App/Root`](#app-root) extension.                   |
+| apis         | The app apis factories.    | [createApiExtension.factoryDataRef](https://backstage.io/docs/reference/frontend-plugin-api.createapiextension.factorydataref)                         | false    | See [default apis](#default-apis-extensions).    | [createApiExtension](https://backstage.io/docs/reference/frontend-plugin-api.createapiextension)     |
+| themes       | The app themes list.       | [createThemeExtension.themeDataRef](https://backstage.io/docs/reference/frontend-plugin-api.createthemeextension.themedataref)                         | false    | See [default themes](#default-theme-extensions). | [createThemeExtension](https://backstage.io/docs/reference/frontend-plugin-api.createthemeextension) |
+| components   | The app components list.   | [createComponentExtension.componentDataRef](https://backstage.io/docs/reference/frontend-plugin-api.createcomponentextension.componentdataref)         | -        | false                                            | See [default components](#defaults-components-extensions).                                           | [createComponentExtension](https://backstage.io/docs/reference/frontend-plugin-api.createcomponentextension)     |
+| translations | The app translations list. | [createTranslationExtension.translationDataRef](https://backstage.io/docs/reference/frontend-plugin-api.createtranslationextension.translationdataref) | -        | false                                            | -                                                                                                    | [createTranslationExtension](https://backstage.io/docs/reference/frontend-plugin-api.createtranslationextension) |
+
+#### Default theme extensions
```

Comment:
Nice! 👍 

This pattern seems pretty sensible. Even if we generate docs at some point for each of these individual extension it's prolly good to have this list of attached extensions to each input